assets/js/amcharts/moonrisekingdom.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright (c) 2018 amCharts (Antanas Marcelionis, Martynas Majeris)
+ *
+ * This sofware is provided under multiple licenses. Please see below for
```

Comment:
please add amcharts lib to package.json

Reply:
> please add amcharts lib to package.json

It's there. Those files are rudiment - forgot to delete them :)
"@amcharts/amcharts4": "^4.10.29"